frontend/tests/integration/components/transportation/study-area-map-test.js

```diff
@@ -4,6 +4,7 @@ import { render, settled, find } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
 import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
 import Component from '@ember/component';
+import { registerEventHandler, triggerEvent }  from '../../../helpers/mapbox/mapbox-stub-helpers';
```

Comment:
I would be rename `triggerEvent` to something else because that is a common ember-test-helpers helper, could be confusing.